cpp/daal/src/algorithms/k_nearest_neighbors/bf_knn_impl.i

```diff
@@ -59,7 +59,8 @@ public:
     services::Status kNeighbors(const size_t k, const size_t nClasses, VoteWeights voteWeights, DAAL_UINT64 resultsToCompute,
                                 DAAL_UINT64 resultsToEvaluate, const NumericTable * trainTable, const NumericTable * testTable,
                                 const NumericTable * trainLabelTable, NumericTable * testLabelTable, NumericTable * indicesTable,
-                                NumericTable * distancesTable)
+                                NumericTable * distancesTable, bf_knn_classification::prediction::internal::PairwiseDistanceType pairwiseDistance,
+                                double minkowskiDegree)
```

Comment:
Shouldn't it be marked as `const` like `k`, `nClasses`?

Reply:
done, thx